### DISCUSSION TITLE
ColorPicker Palette

### DISCUSSION THREAD

**Discussion by drei97** [AvaloniaUI/Avalonia#16823] [Q&A] [answered]
Hi,

I want to use the ColorPicker control but I cannot find information how the color palettes are named. In the documentation

[https://docs.avaloniaui.net/docs/reference/controls/colorpicker/](url)

they are shown but not named.

```
<ColorPicker Grid.Column="1" Grid.Row="4" 
             Color="{Binding ColorSelected}" 
             Palette=" ... ??? ..." />

```

Joachim

**Comment by stevemonaco** [comment, marked as answer]
In these cases, you should search the Avalonia repo for relevant topics. ie. `IColorPalette`. You'll find them in source here: https://github.com/AvaloniaUI/Avalonia/tree/06ece8da8813119b0feb0d35de397dd031157095/src/Avalonia.Controls.ColorPicker/ColorPalettes

You can/should also post an issue to the Avalonia docs repo.
